lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -881,9 +882,9 @@ def process(self, mean_values, variance_values, probability_cube_template):
                 Cube containing the distribution variance values of a
                 diagnostic, e.g. the variance across realizations.
             probability_cube_template (iris.cube.Cube):
-                A probability cube that has the threshold coordinate, and
-                attribute relative_to_threshold, that match the desired output
-                cube format.
+                A probability cube that has a threshold coordinate with an
+                spp__relative_to_threshold attribute that matches the desired
+                output cube format.
```

Comment:
As above.

Reply:
I've updated the docstring as suggested.